solr/solrj/src/java/org/apache/solr/common/cloud/ClusterState.java

```diff
@@ -425,7 +425,10 @@ public Stream<DocCollection> collectionStream() {
   /**
    * Calls {@code consumer} with a resolved {@link DocCollection}s for all collections. Use this
    * sparingly in case there are many collections.
+   *
+   * @deprecated see {@link #collectionStream()}
```

Comment:
If this method was introduced a few days ago (can't remember), I think it can be removed rather than deprecated.

Reply:
It has existed for some releases, thus I deprecate it.